### DISCUSSION TITLE
Unable to deploy Share Extension to iOS simulator

### DISCUSSION THREAD

**Discussion by bptouchette** [VladislavAntonyuk/MauiSamples#207] [Q&A] [answered]
I've been trying to make the iOS share extension work with maui for a while now but encountered a road block when it comes to deploying the share extension to an iOS simulator.

I'm using Visual Studio 2022 17.5.0 Preview 3
Paired to a Mac with Xcode 14.2
Visual Studio for Mac latest version
Simulator : iPad (10th generation) iOS 16.2

I've tried my own application, that didn't work.
I've tried the JokeApp project from the MauiSamples and I get the same problem.
I've tried the AppContainer project from the MauiSamples and I get the same problem.

2>------ Deploy started: Project: JokeAppWidget, Configuration: Debug Any CPU ------
2>Please select a valid device before running the application.
2>Deploy failed
3>------ Deploy started: Project: JokeApp (iOSExtensions\JokeApp\JokeApp), Configuration: Debug Any CPU ------
========== Build: 1 succeeded, 0 failed, 1 up-to-date, 0 skipped ==========
========== Build started at 1:35 PM and took 35,824 seconds ==========
========== Deploy: 1 succeeded, 1 failed, 0 skipped ==========
========== Deploy started at 1:35 PM and took 35,824 seconds ==========

**Comment by VladislavAntonyuk** [comment, marked as answer]
JokeAppWidget is experimental and currently doesn't work as expected. AppContainer should work fine.

**Reply by bptouchette** [reply]
Unfortunately, I can't make AppContainer work either. Everything builds fine by themselves but I can't seem to deploy to the device.

**Reply by VladislavAntonyuk** [reply]
Can you try it on Mac?

**Reply by bptouchette** [reply]
After trying on Visual studio for Mac, I've been able to make AppContainer work and my own app work. 
It will be a good plan B for now but I would of love to see this work on Visual Studio on Windows.
I still don't know what's causing the issue. If anyone finds anything I would love to know about it.
Thank you for the help.

**Reply by VladislavAntonyuk** [reply]
You can create an issue in dotnet/maui repository
